examples/large_package_publisher.py

```diff
@@ -64,6 +64,8 @@ async def main(
         alice, large_package_example_dir, large_packages_account
     )
 
+    await rest_client.close()
```

Comment:
Nit, can we use the client as a context manager? The indentation is annoying though...